internal/renderers/femtovg/itemrenderer.rs

```diff
@@ -1303,39 +1276,31 @@ impl<'a> GLItemRenderer<'a> {
     fn draw_image_impl(
         &mut self,
         item_rc: &ItemRc,
-        source_property: Pin<&Property<Image>>,
-        source_clip_rect: IntRect,
-        target_width: Pin<&Property<LogicalLength>>,
-        target_height: Pin<&Property<LogicalLength>>,
-        image_fit: ImageFit,
-        colorize_property: Pin<&Property<Brush>>,
-        image_rendering: ImageRendering,
+        item: Pin<&dyn RenderImage>,
```

Comment:
This could be called `image`, now that it's not an item anymore :)

Reply:
Yes, that's how i called it at first.  But there is another variable called image in that function so that won't work.
And it's still an item. It's an Item that implements the RenderImage trait.